app/views/conversations/_show.mobile.haml

```diff
@@ -2,16 +2,21 @@
 -# licensed under the Affero General Public License version 3 or later. See
 -# the COPYRIGHT file.
 
-.conversation_participants
-  .right
-    = link_to(raw("<i class='entypo trash'></i>"), conversation_visibility_path(conversation),
-            method: 'delete', data: { confirm: "#{t('.delete')}?" }, class: "remove")
+.conversation
+  .conversation_participants.header-full-width
+    .delete_conversation.right
+      = link_to(raw("<i class='entypo trash'></i>"), conversation_visibility_path(conversation),
+              method: 'delete', data: { confirm: "#{t('.delete')}?" }, class: "remove")
 
-  %h3{ class: direction_for(conversation.subject) }
-    = conversation.subject
+    %h3
+      = conversation.subject
+    = timeago(conversation.updated_at)
 
-  - for participant in conversation.participants
-    = person_image_link(participant, size: :thumb_small)
+    %div
+      - for participant in conversation.participants
+        .avatar
+          = person_image_link(participant, size: :thumb_small)
+    .clear
 
-.span-15.last
-  = render partial: 'messages', locals: { conversation: conversation }
+  .span-15.last
```

Comment:
That has no effect anymore. Remove it and use a simple `%div` instead or use Bootstrap's grid.